Bangazon/Bangazon/urls.py

```diff
@@ -6,6 +6,7 @@
 # Create a router, using DefaultRouter
 # Register each ViewSet with it.
 router = DefaultRouter()
+router.register(r'staff', views.UserStaffViewSet)
```

Comment:
No need for it actually...Since UserProfileView Is all we need for the url. I'll delete that part and push again...good catch(It wasn't supposed to be there and pushed)